src/pool/user.c

```diff
@@ -307,6 +307,10 @@ pool_user_login(TDS_POOL * pool, TDS_POOL_USER * puser)
 	}
 
 	puser->login = login = tds_alloc_login(true);
+	if (!login) {
+		fprintf(stderr, "tds_alloc_login() failed.\n");
```

Comment:
I would replace `fprintf` with a log. Can I change it?

Reply:
yep